core/esmf-aspect-model-validator/src/main/java/org/eclipse/esmf/aspectmodel/shacl/RustLikeFormatter.java

```diff
@@ -231,6 +231,12 @@ private boolean formatNode( final RDFNode node ) {
          if ( node.asNode().equals( NodeConst.nodeRDFType ) ) {
             spacedIfPossible( "a " );
          }
+         if ( node.asNode().equals( NodeConst.nodeTrue ) ) {
```

Comment:
I would recommend to use the constant first 
NodeConst.nodeTrue.equals(node.asNode())